src/wp-includes/widgets/class-wp-nav-menu-widget.php

```diff
@@ -149,8 +161,8 @@ public function form( $instance ) {
 				</select>
 			</p>
 			<?php if ( $wp_customize instanceof WP_Customize_Manager ) : ?>
-				<p class="edit-selected-nav-menu" style="<?php if ( ! $nav_menu ) { echo 'display: none;'; } ?>">
-					<button type="button" class="button"><?php _e( 'Edit Menu' ) ?></button>
+				<p class="edit-selected-nav-menu" style="<?php echo $nav_menu_style; ?>">
```

Comment:
This should remain inline instead of setting a variable.
And it should not output `style=` in the case of a nav menu.

Reply:
I see what you mean, however this is duplicated from upstream so issue also in WP.

Reply:
The other ones in the same file have the style keyword as part of the computed string.

Reply:
Discussed at PR review.